content/guides/models/app/settings-page/user-settings.md

```diff
@@ -16,6 +16,21 @@ Navigate to your user profile page and select your user icon on the top right co
 
 Within the **Profile** section you can manage and modify your account name and institution. You can optionally add a biography, location, link to a personal or your institution’s website, and upload a profile image.
 
+## Edit your intro
+
+To edit your intro, click **Edit** at the top of your profile. The WYSIWYG editor that opens supports Markdown.
+1. To edit a line, click it. To save time, you can type `/` and choose Markdown from the list.
+1. Use an item's drag handles to move or resize it.
+1. To save your changes, click **Save**.
+
+### Add social badges
+
+To add a follow badge for the `@weights_biases` account on X, you could add a Markdown-style link with an HTML `<img>` tag that points to the badge image:
+
+```markdown
+[<img src="https://img.shields.io/twitter/follow/weights_biases?style=social" alt="X: @weights_biases" >](https://x.com/intent/follow?screen_name=weights_biases)
+```
+
```

Comment:
```suggestion
```

Don't think you meant to include this here.

Reply:
I did actually, before this change, the Markdown example here is broken and shows up as a "broken" link. _And_ the content itself is in the wrong page, it's a User Profile setting, not a Team setting. This change fixes the code and puts it in the right place.